code/go/0chain.net/blobbercore/allocation/protocol.go

```diff
@@ -132,6 +134,13 @@ func VerifyAllocationTransaction(ctx context.Context, allocationTx string, reado
 	a.TimeUnit = sa.TimeUnit
 	a.IsImmutable = sa.IsImmutable
 
+	m := map[string]interface{}{
+		"allocation_id":  a.ID,
+		"allocated_size": (sa.Size + int64(len(sa.BlobberDetails)-1)) / int64(len(sa.BlobberDetails)),
+	}
+
+	go filestore.GetFileStore().UpdateAllocationMetaData(m)
```

Comment:
It will ignore update if there is type mismatch. Otherwise it won't fail.
```
        m := map[string]interface{}{
		"allocation_id":  a.ID,
		"allocated_size": (sa.Size + int64(len(sa.BlobberDetails)-1)) / int64(len(sa.BlobberDetails)),
	}
```

With this map it will never fail.

One solution for this would be to avoid using go routine and return error, should it fail to update.
Do you suggest it?

Reply:
Yes, I suggest to do it without goroutine if it is not a resources heavy process.